### PR TITLE
fix build hang

### PR DESCRIPTION
**Pull request by lukescott**
From what I could tell node.stop() was never called. I fixed that, and added some active() methods for a test I added to use to determine if node was still active or not.

Also had to rework the test/build.js to be similar to test/scenarios.js as there were some race conditions with the tests in that file.

Hopefully this hasn't broken anything else, but all the tests pass.

Related to gobblejs/gobble-cli/pull/11 and gobblejs/gobble-cli/issues/12.

### CODE REVIEW

File: src/nodes/build/index.js [hunk 1]
```diff
@@ -39,7 +39,8 @@ export default function ( node, options ) {
 			.then(
 				inputdir => copydir( inputdir ).to( dest ),
 				err => { throw err; }
-			);
+			)
+			.then(() => node.stop());
```
Comment (evs-chris) [inline src/nodes/build/index.js]: d'oh... I added the start, but didn't think to stop it. :+1:

File: test/build.js [hunk 2]
```diff
@@ -1,34 +1,40 @@
...
-
-			// Clean up output dir
-			return sander.rimraf( 'output' );
+			return sander.rimraf( 'tmp' );
 		});
 
-		it( 'should return a promise that fulfils on completion of build', function () {
-			return require( './sample/foo' ).build({
-				dest: output
+		it( 'should return a promise that fulfills on completion of build', function () {
```
Comment (Rich-Harris) [inline test/build.js]: ಠ_ಠ [those pesky americans!](http://grammarist.com/spelling/fulfil-fulfill/)

Comment (lukescott) [inline test/build.js]: Do'h! Sorry, I have spellcheck on in Sublime :).

**Comment by Rich-Harris** [comment]
This might also be fixed by the solution to https://github.com/gobblejs/gobble/issues/50, but since that's a 'might' and this is a 'does', in it goes! Thanks @lukescott

**Comment by Rich-Harris** [comment]
Released gobble 0.9.1 alongside 0.4.0. Should be good now

**Comment by lukescott** [comment]
Awesome, just updated my project and it works! 24 line build file (10 of it banner), so cool! Thanks guys.

**Pull request merged**